app/scripts/modules/core/src/application/listExtractor/listExtractor.service.ts

```diff
@@ -15,6 +16,13 @@ const defaultFilter = () => true;
 
 export class AppListExtractor {
 
+  public getMonikers(applications: Application[], filter: IServerGroupFilter = defaultFilter): Moniker[] {
```

Comment:
I would omit the `filter` here. If it turns out it's needed later, we can always add it back in.

Reply:
@anotherchrisberry - we are using the filter on line 21.  could you clarify your statements above?

Reply:
Sorry, I would omit it because you're not calling `.getMonikers` with two arguments anywhere - just `appListExtractorService.getMonikers([$scope.application])`. Unless you're expecting to supply that second argument in the near future for this, there's no sense in including it here IMO.

Reply:
makes sense.  Let me make that modification now.